packages/dd-trace/src/appsec/channels.js

```diff
@@ -1,12 +1,19 @@
 'use strict'
 
 const dc = require('dc-polyfill')
+const { channel } = require('../../../datadog-instrumentations/src/helpers/instrument')
 
 // TODO: use TBD naming convention
 module.exports = {
   bodyParser: dc.channel('datadog:body-parser:read:finish'),
   cookieParser: dc.channel('datadog:cookie-parser:read:finish'),
-  graphqlFinishExecute: dc.channel('apm:graphql:execute:finish'),
+  startGraphqlResolve: dc.channel('apm:graphql:resolve:start'),
+  startGraphqlMiddleware: dc.channel('datadog:apollo:middleware:start'),
+  endGraphqlMiddleware: dc.channel('datadog:apollo:middleware:end'),
+  startExecuteHTTPGraphQLRequest: dc.channel('datadog:apollo:request:start'),
+  startGraphqlWrite: dc.channel('datadog:apollo:request:success'),
+  startApolloServerCoreRequest: channel('datadog:apollo-server-core:request:start'),
+  successApolloServerCoreRequest: channel('datadog:apollo-server-core:request:success'),
```

Comment:
You don't need to share channels like this. The name acquires the channel from a global namespace already. If anything, it's best _not_ to share channels like this because this way will create a whole bunch of extra channels even if they're never used.

Reply:
You're right. We are planning to refactor this part in a following PR